xml/ay_bigfile.xml

```diff
@@ -5241,6 +5241,37 @@ config:type="boolean"&gt;true&lt;/keep_install_network&gt;</screen>
 &lt;/networking&gt;</screen>
    </example>
 
+   <example>
```

Comment:
It is generally a good idea to add `xml:id`s to formal elements that have a title (procedure, example, figure, ...)